src/Marten/LinqExtensions.cs

```diff
@@ -19,6 +19,18 @@ public static bool IsOneOf<T>(this T variable, params T[] matches)
             return matches.Contains(variable);
         }
 
+        /// <summary>
+        /// Used for Linq queries to match an element to one of a list of values
+        /// </summary>
+        /// <typeparam name="T"></typeparam>
+        /// <param name="variable"></param>
+        /// <param name="matches"></param>
+        /// <returns></returns>
+        public static bool IsOneOf<T>(this T variable, IEnumerable<T> matches)
```

Comment:
**FIX:** This code won't work if you pass IEnumerable that's not Array or List. Npgsql will throw the following error: `System.NotSupportedException: 'Npgsql 3.x removed support for writing a parameter with an IEnumerable value, use .ToList()/.ToArray() instead'`. I'd suggest adding overload with IList instead of IEnumerable.

Reply:
Will do.